### PR TITLE
ziro: layouting fix

### PR DESCRIPTION
**Pull request by fortoszone**
some fix for ziro
Screen Shot 2022-07-28 at 23 08 45
Screen Shot 2022-07-28 at 23 07 26

### CODE REVIEW

File: Ziro/user.css [hunk 1]
```diff
@@ -517,3 +517,10 @@ input:checked ~ .x-toggle-indicatorWrapper .x-toggle-indicator {
 .main-rootlist-rootlistItemOverlay {
   display: none;
 }
+.main-rootlist-rootlist {
+  --left-sidebar-padding-left: 8px;
+  --left-sidebar-padding-right: 8px;
+}
+label.Type__TypeElement-goli3j-0.gCwing.main-playlistEditDetailsModal-textElementLabel {
+  padding-bottom: 12px;
+}
```
Comment (kyrie25) [inline Ziro/user.css:526]: What exactly does this do?

Comment (kyrie25) [reply Ziro/user.css:526]:
```suggestion
.main-playlistEditDetailsModal-textElementLabel {
  padding-bottom: 12px;
}
```

**Comment by fortoszone** [comment]
sorry for the delay. i made more fix for the theme, mainly aligning and change color on the navbar icon
Screenshot 2022-11-01 at 16 58 09

Spotify for macOS (Intel)
1.1.97.962.g24733a46
Spicetify v2.14.1

**Comment by harbassan** [comment]
Do these changes still work?

**Comment by fortoszone** [comment]
Yes, i think it works still

Spotify for macOS (Intel)
1.1.97.962.g24733a46
Spicetify v2.14.3

Screenshot 2022-12-15 at 08 49 27

**Comment by fortoszone** [comment]
Newer than 1.1.97.962 since i only tried on that version

**Review by harbassan — APPROVED** [review]

**Pull request merged**